server/controllers/comments.js

```diff
@@ -44,43 +86,109 @@ const addCommentToPost = async (req, res) => {
   }
 };
 
+// @route   POST '/api/posts/:postId/comments/:commentId/replies'
+// @desc    add a reply to a specific comment
+// @access  private
+const addReplyToComment = async (req, res) => {
+  try {
+    const post = await getPost(req.params.postId);
+    const comment = await getComment(req.params.commentId);
+
+    if (!comment) {
+      return serverResponse(res, 404, { message: "Comment doesn't exist" });
+    }
+
```

Comment:
move `const post = await getPost(req.params.postId);` to line 100 because if there is no comment you dont need to fetch the post